PowerFGT/Public/cmdb/firewall/address.ps1

```diff
@@ -209,17 +219,16 @@ function Get-FGTFirewallAddress {
         [string]$name,
         [Parameter (Mandatory = $false, ParameterSetName = "uuid")]
         [string]$uuid,
-        [Parameter (Mandatory = $false)]
-        [Parameter (ParameterSetName = "filter")]
+        [Parameter(Mandatory = $false, ParameterSetName = "filter")]
+        [String]$filter,
```

Comment:
i don't really like the idea to have directly filter parameter.. but prefer we can build filter with filter_type, attribute...

Reply:
Oke i will remove it. Advanced searches can be done with invoke-FGTRestMothode if needed. is the parameter name-change  (filter -> filter_build) oke?  This will make it consistant with Invoke-FGTRestMethode. If not i will change it back.

Reply:
Ok for change ParameterName